ns/list.html

```diff
@@ -148,7 +148,9 @@ <h5>Property Definitions</h5>
 If the object is a variable, it will create a solution for each member of the subject list.
 If any member of the object list is a variable, it will create a solution for all matching
 members of the subject list.
-If the object is ground, it evaluates to <code>true</code>; otherwise, <code>false</code>.</p>
+If the object is ground, it evaluates to <code>true</code>
+if the entry at the specified index matches the specified value;
+otherwise, <code>false</code>.</p>
```

Comment:
```suggestion
If the object is ground and the entry at the specified index matches the specified value, 
it evaluates to <code>true</code>; otherwise, <code>false</code>.</p>
```